uint8 movement_command
uint8 cam_angle_command